app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -23,24 +23,29 @@ def fetch_hawk_inv(ems)
         return nil
       end
 
-      def parse_legacy_inventory(ems)
+      # Full refresh. Collecting immediately. Don't have separate Collector classes.
+      def collect_inventory_for_targets(ems, _targets)
         request_entities = OPENSHIFT_ENTITIES.dup
         request_entities << {:name => 'images'} if refresher_options.get_container_images
 
-        kube_entities = ems.with_provider_connection(:service => KUBERNETES_EMS_TYPE) do |kubeclient|
+        kube_inventory = ems.with_provider_connection(:service => KUBERNETES_EMS_TYPE) do |kubeclient|
           fetch_entities(kubeclient, KUBERNETES_ENTITIES)
         end
-        openshift_entities = ems.with_provider_connection do |openshift_client|
+        openshift_inventory = ems.with_provider_connection do |openshift_client|
           fetch_entities(openshift_client, request_entities)
         end
-        entities = openshift_entities.merge(kube_entities)
-        entities["additional_attributes"] = fetch_hawk_inv(ems) || {}
-        EmsRefresh.log_inv_debug_trace(entities, "inv_hash:")
 
+        inventory = openshift_inventory.merge(kube_inventory)
```

Comment:
maybe merge!, there can be quite big hashes, so this was we can reuse the openshift_inventory?